docs/administration/github-org.rst

```diff
@@ -1,10 +1,102 @@
+###################
 GitHub organization
-===================
+###################
 
 This page explains how the `RITlug GitHub organization`_ is organized and how
-it is intended to be used.
+to manage it.
 
-.. _`RITlug GitHub organization`: https://github.com/RITlug
 
-…more to come soon.
+*******
+Purpose
+*******
+
+The GitHub organization is a central place to manage club projects, membership,
+and infrastructure. Most data is static and doesn't change often. Setting up
+club projects or adding a new member is when you usually need to interact with
+the GitHub organization.
+
+
+************
+Repositories
+************
+
+There are no restrictions on repositories allowed. Repository creation
+permission is granted only to eboard members as a precaution. This approach was
+decided to ensure that the current eboard has a chance to review a project
+before officially endorsing it underneath our GitHub organization.
+
+Generally, repository creation should be encouraged rather than discouraged.
+
+
+******************
+Adding new members
+******************
+
+There is no criteria for adding new members to the RITlug GitHub organization.
```

Comment:
Really? This also contradicts the "active community members" on L39.

Reply:
@ct-martin When I said we don't have criteria for adding new members, we do not have a firm set of criteria, e.g. attend X number of meetings, participate over X amount of time, etc.

What change are you suggesting?

Reply:
@jflory7 At the very least, not saying there are no criteria. If I remember correctly, the "technical" definition for Club Office is attending three meetings/semester (correct me if I'm wrong). While this may seem like an unlikely scenario, what we don't want to have happen is someone who none of us have ever interacted with coming and waving this in our face.

Reply:
@ct-martin I think this fits into a larger discussion for our club membership. I am not aware of any existing guideline / requirement even to club office for our membership. As it is now, there really is not any criteria. I only add people when I remember.

We can move a membership discussion to [RITlug/tasks](https://github.com/RITlug/tasks), but I don't think this should block this PR. We can go back and make a change after we consider general membership criteria.

Personally, I would like club members to ask to be included in the GitHub organization. In your scenario you gave, we can use our own discretion for how to answer. If someone who we never see asks to be included in our GitHub organization, I would not consider them an active community member and would use that as my reply.

Reply:
@jflory7 In that case, I would put being an active community member as the criteria, or just drop the line in favor of L39, "Active community members of RITlug should be added to the GitHub organization."
I'm going to maintain my blocking review and cite your response to my scenario as additional reasoning for why.